dashboard/src/actions/comparisonActions.js

```diff
@@ -139,32 +184,37 @@ export const parseChartData = () => (dispatch, getState) => {
   });
 };
 
-export const toggleCompareSwitch = () => (dispatch, getState) => {
-  dispatch({
-    type: TYPES.TOGGLE_COMPARE_SWITCH,
-    payload: !getState().comparison.isCompareSwitchChecked,
-  });
-};
+export const toggleCompareSwitch = () => ({
+  type: TYPES.TOGGLE_COMPARE_SWITCH,
+});
```

Comment:
If you omit the trailing comma on line 188, I think this will all fit on one line.